workspace.json

```diff
@@ -244,6 +244,12 @@
                         "cypressConfig": "apps/e2e/cypress.config.ts",
                         "tsConfig": "apps/e2e/tsconfig.json",
                         "testingType": "e2e"
+                    },
+                    "configurations": {
+                        "ci": {
+                            "headed": true,
+                            "browser": "chrome"
+                        }
```

Comment:
There was one test that was having issues running in headless electron mode, but ran fine on my machine. Running the tests headed on Chrome fixed it. Some discussion here [https://github.com/cypress-io/cypress/issues/8606#issuecomment-703544094](https://github.com/cypress-io/cypress/issues/8606#issuecomment-703544094)